task/pnc-prebuild-git-clone-oci-ta/0.1/patch.yaml

```diff
@@ -0,0 +1,71 @@
+- op: replace
+  path: /metadata/name
+  value: pnc-prebuild-git-clone-oci-ta
+- op: replace
+  path: /metadata/annotations/tekton.dev~1displayName
+  value: pnc prebuild git clone oci trusted artifacts
+- op: replace
+  path: /spec/description
+  value: The pnc-prebuild-git-clone-oci-ta Task will clone a repo from the provided url, apply
+    PNC prebuild modifications from https://github.com/project-ncl/konflux-tooling and store
```

Comment:
`konflux-tooling` sounds too generic for me, maybe `pnc-konflux-tooling` would be better?

Reply:
I am not sure how we can change the image name as isn't it derived from the repository name?

Reply:
You can release it via Konflux